Gopkg.toml

```diff
@@ -45,35 +45,35 @@ required = [
   name = "github.com/docker/distribution"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
   name = "github.com/giantswarm/apiextensions"
 
 [[constraint]]
-  branch = "master"
+  branch = "kubernetes-1.15.4"
```

Comment:
This is to be reverted once https://github.com/giantswarm/apprclient/pull/26 gets merged.